src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericDateInput.js

```diff
@@ -41,7 +41,14 @@ GenericDateInput.propTypes = {
   parameterValue: PropTypes.any,
   setParameterValue: PropTypes.func.isRequired,
   isDirty: PropTypes.bool,
+  error: PropTypes.object,
 };
 GenericDateInput.defaultProps = {
   isDirty: false,
 };
+GenericDateInput.useValidationRules = (parameterData) => {
```

Comment:
`parameterData` seems not to be used in this hook

Reply:
yes, it's not used for now, it's here because we will certainly need it for further validations, I put it here for future tickets :slightly_smiling_face:

Reply:
I see. It feels like it would be better to add it later (in a future PR). If you think it's important to leave a note right now in the code, maybe a comment would be enough?